homework3/semi-uber/UI/src/app/regestration-page/regestration-page.component.ts

```diff
@@ -21,12 +28,10 @@ export class RegestrationPageComponent implements OnInit {
       Validators.required,
       Validators.pattern(this.passwordRegexp)
     ]),
-    email: new FormControl("", [
-      Validators.required,
-      Validators.pattern(this.emailRegexp)
-    ])
+    email: new FormControl("", [Validators.required, Validators.email]),
```

Comment:
Now you can remove `emailRegexp `. You do not need it anymore

Reply:
done